### ISSUE TITLE
Imgbot not running on a public repo

### ISSUE THREAD

**Issue by JakeSteam** [imgbot/Imgbot#1336]
Hey! 

I've seen #757, but I believe my issue is distinct.

I have 2 very similar projects, both with hundreds of images, and imgbot works perfectly on one and not at all on the other.

* [Project 1](https://github.com/JakeSteam/blog-programming): Open source jekyll blog, imgbot added with no config, new imgbot PR raised every week or two.
* [Project 2](https://github.com/JakeSteam/blog-personal): Open source jekyll blog, imgbot added with no config, no imgbot PRs ever raised.

I tried requesting an optimisation via imgbot.net 19 hours ago, yet nothing seems to have happened:
![image](https://github.com/imgbot/Imgbot/assets/12380876/969c5fd3-d412-465b-a11c-71c42906f58e)

Any clues?

**Comment by JakeSteam** [comment]
As an update, 3 weeks later imgbot still hasn't run, despite a few hundred unoptimized images 😅 

https://github.com/JakeSteam/blog-personal/tree/main/assets/images/2023

**Comment by maybeanerd** [comment]
For me , it did finally Run after a week of waiting. Not helpful to you I know, Just wanted to let you know the service is not entirely dead

**Comment by JakeSteam** [comment]
@maybeanerd Yep, it works perfectly on one of those two repos, and not on the very similar second repo! It's been over a year, so I don't think it will fix itself..!

**Comment by JakeSteam** [comment]
I eventually gave up on using imgbot on this repo, since it still did nothing after being removed & readded. I decided to optimised manually, and now suspect the problem was in an underlying dependency, since I had issues with other solutions too!

The repository had 1.5k+ images to optimise, and 186MB of savings, here's the `.yml` I eventually used to manually optimise: https://github.com/calibreapp/image-actions/issues/118#issuecomment-1950677701

**Comment by danner26** [comment]
Honestly it looks like this repo is stale. I wish I could get into contact with the owner, I would love to take over it and bring it back to what it was
